### DISCUSSION TITLE
How to disable a specific deptype in maven?

### DISCUSSION THREAD

**Discussion by Lynnes001** [renovatebot/renovate#26432] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### What is your question?

i am using maven and it's a multi-module project. I saw there is a `matchDepType` option to specify types. But my renovate send a PR with type `parent` which i want to ignore. Is there a way that to ignore a DepType? Or what is the full list of depTypes? Thanks.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by viceice** [comment]
yes, you can use package rules

**Reply by Lynnes001** [reply]
Hi @viceice, would you please guide which property i should use? I only find [matchDepType](https://docs.renovatebot.com/configuration-options/#matchdeptypes) but not something like `excludeDepType`?

**Reply by rarkins** [reply, marked as answer]
matchDepTypes=foo, enabled=false
